rocAL/rocAL/source/augmentations/node_ssd_random_crop.cpp

```diff
@@ -49,12 +47,12 @@ void SSDRandomCropNode::create_node()
         THROW("Uninitialized destination dimension")
 
     _crop_param->create_array(_graph);
-    // _node = vxExtrppNode_CropPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _crop_param->cropw_arr,
-                                // _crop_param->croph_arr, _crop_param->x1_arr, _crop_param->y1_arr, _batch_size);
+    create_crop_tensor(_crop_tensor, &_crop_coordinates);
+    _node = vxRppCrop(_graph->get(), _inputs[0]->handle(), _crop_tensor, _outputs[0]->handle(), _input_layout, _output_layout, _roi_type);
 
     vx_status status;
     if ((status = vxGetStatus((vx_reference)_node)) != VX_SUCCESS)
-        THROW("Error adding the crop resize node (vxExtrppNode_ResizeCropbatchPD    ) failed: " + TOSTR(status))
+        THROW("Error adding the crop resize node (vxExtrppNode_ResizeCropbatchPD) failed: " + TOSTR(status))
```

Comment:
Change 
THROW("Error adding the crop resize node (vxExtrppNode_ResizeCropbatchPD) failed: " + TOSTR(status))
to 
THROW("Error adding the crop resize node (vxRppCrop) failed: " + TOSTR(status))


Reconfirm once in other files too